.vscode/tasks.json

```diff
@@ -1,17 +1,64 @@
 {
-    "version": "0.1.0",
-    "command": "pwsh",
-    "isShellCommand": true,
-    "showOutput": "always",
-    "suppressTaskName": true,
-    "args": [ "-command" ],
+    "version": "2.0.0",
+
+    "windows": {
+        "options": {
+            "shell": {
+                "executable": "pwsh.exe",
+                "args": [
+                    "-NoProfile",
+                    "-ExecutionPolicy",
+                    "Bypass",
+                    "-Command"
+                ]
+            }
+        }
+    },
+    "linux": {
+        "options": {
+            "shell": {
+                "executable": "/usr/bin/pwsh",
+                "args": [
+                    "-NoProfile",
+                    "-Command"
+                ]
+            }
+        }
+    },
+    "osx": {
+        "options": {
+            "shell": {
+                "executable": "/usr/local/bin/pwsh",
+                "args": [
+                    "-NoProfile",
+                    "-Command"
+                ]
+            }
+        }
+    },
 
     "tasks": [
         {
-            "taskName": "build",
-            "args": [ "Import-Module ${workspaceRoot}/build.psm1; Start-PSBuild -Output ${workspaceRoot}/debug" ],
-            "isBuildCommand": true,
+            "label": "Bootstrap",
+            "type": "shell",
+            "command": "Import-Module ${workspaceFolder}/build.psm1; Start-PSBootstrap",
+            "problemMatcher": []
+        },
+        {
+            "label": "Clean Build",
+            "type": "shell",
+            "command": "Import-Module ${workspaceFolder}/build.psm1; Start-PSBuild -Clean",
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build",
+            "type": "shell",
+            "command": "Import-Module ${workspaceFolder}/build.psm1; Start-PSBuild -Output (Join-Path ${workspaceFolder} debug)",
```

Comment:
Nice!  Can you add the same parameter to the `Clean Build` task above?  Also, `Clean Build` reads a little awkward.  What if we called this task `Rebuild`?  On VS at least, that implies Clean, then Build.

Reply:
BTW I was just going to warn you about this issue when I saw you not only already knew about it but solved it.  👍  Also, I tagged onto the VSCode issue on the $mscompile problem matcher not working - https://github.com/Microsoft/vscode/issues/34660

Reply:
Will add to `Clean Build` which I think is named fine

Reply:
@SteveL-MSFT I got a response from the VSCode team on the $mscompile issue.  All we need to do is add the command line arg - `/property:GenerateFullPaths=true`.  I tried putting this around line 485:
```
    $Arguments += "/property:GenerateFullPaths=true"
```
And it works!  There may be other places where this arg needs to be provided.

Reply:
Will add.  Thanks!